Assets/MixedRealityToolkit.Extensions/SpectatorView/Plugins.meta

```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 5e04ee94a48a1bf419f04efab59156e8
+guid: 6c6aaecea752d46499dccbf44ddfecb8
 folderAsset: yes
 DefaultImporter:
```

Comment:
I don't know that we should have any plugins directly checked in for this project. We should probably delete this directory and use nuget packages/nuget for unity folders

Reply:
Not sure why I have a change here.